android/KMEA/app/src/main/java/com/keyman/engine/KMKeyboard.java

```diff
@@ -381,23 +363,12 @@ public boolean onTouchEvent(MotionEvent event) {
     // Come to think of it, I wonder if suggestionMenuWindow was work being done to link with
     // suggestion banner longpresses - if so, it's not yet ready for proper integration...
     // and would need its own rung in this if-else ladder.
-    if (subKeysWindow != null && suggestionMenuWindow == null && subKeysWindow.isShowing()) {
-      // Passes KMKeyboard (subclass of WebView)'s touch events off to our subkey window
-      // if active, allowing for smooth, integrated gesture control.
-      subKeysWindow.getContentView().findViewById(R.id.grid).dispatchTouchEvent(event);
-    } else {
-      if (event.getPointerCount() > 1) {
-        // Multiple points touch the screen at the same time, so dismiss any pending subkeys
-        dismissKeyPreview(0);
-        dismissSubKeysWindow();
-      }
+    if (true) {
       gestureDetector.onTouchEvent(event);
     }
```

Comment:
```suggestion
    gestureDetector.onTouchEvent(event);
```

But do we even need `gestureDetector` any more?